app/src/ui/check-runs/ci-check-run-list-item.tsx

```diff
@@ -63,21 +85,32 @@ export class CICheckRunListItem extends React.PureComponent<
               title={checkRun.description}
             />
           </div>
-
           <div className="ci-check-list-item-detail">
             <div className="ci-check-name">{checkRun.name}</div>
             <div className="ci-check-description" title={checkRun.description}>
               {checkRun.description}
             </div>
           </div>
+          <div
+            className={classNames('view-on-github', {
+              show: this.state.isMouseOverLogs,
+            })}
+            onClick={this.onViewOnGitHub}
+          >
+            <Octicon
+              symbol={OcticonSymbol.linkExternal}
+              title="View on GitHub"
+            />
+          </div>
```

Comment:
Setting up the tooltip only for the octicon feels weird (especially if you add the `:hover` color change to `.view-on-github` as I suggest in my other comment) because it's smaller than the parent `div`, and clicking on that `div` but outside of the `Octicon` will open GitHub even if you don't see the tooltip.

I tried just replacing the `div` with a `TooltippedContent` element but then the tooltip doesn't appear from the center of the `div`/octicon 😢  It would probably require some extra work to make all this look 100% perfect… what do you think?

Reply:
I don't know.. The tooltip feels a bit aggressive to me already.. I even considered not having it. IMO, it would feel weird for it to popup when I wasn't on the icon (even tho the click area is a bit bigger which makes it easy to hit).

Reply:
After discussion, have changed the text color change to when you are over the icon so the text color changes feels in conjunction with the tool tip.